README.md

```diff
@@ -422,6 +422,11 @@ your development environment is likely a good starting point:
     cp .env .env.integration
 
 
+There is also a cleanup routine intended for use by CI services to check for and cleanup any dangling OpenStack VMs past a certain age threshold. While it isn't necessary in the usual case, old integration tests that were killed without cleanup can be cleaned up after four hours by running the make target:
```

Comment:
@bdero "to check for and cleanup" -- should that be "to check for and clean up"?